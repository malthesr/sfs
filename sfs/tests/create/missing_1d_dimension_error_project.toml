args = ["create",
  "-vv",
  "-p",
  "5,3",
  "tests/create/missing.bcf"
]
status = "failed"
