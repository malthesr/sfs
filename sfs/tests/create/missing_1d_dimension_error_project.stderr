cannot project from one number of dimensions (1) to another (2)
